utils/releaser/src/main.rs

```diff
@@ -11,10 +11,16 @@ use crate::secrets::*;
 use crate::utils::root;
 
 use structopt::StructOpt;
+use utils::bump_versions;
 
 #[derive(PartialEq, StructOpt)]
+#[structopt(name = "basic")]
 enum Releaser {
-    All,
+    All {
```

Comment:
This needs to be it's own command, `All` gets executed on a handful of machines at the same time